### DISCUSSION TITLE
Is it possiable to login to with token but not account and passwds

### DISCUSSION THREAD

**Discussion by someone132s** [vladimirs-git/fortigate-api#35] [Q&A]
as above

**Comment by vladimirs-git** [comment]
Yes, it is possible to log in using a token instead of an account and password.
These resources provide detailed information and examples on how to implement token-based authentication.
https://fortigate-api.readthedocs.io/en/latest/fortigateapi/FortiGateAPI.html#fortigate_api.FortiGateAPI
https://github.com/vladimirs-git/fortigate-api/blob/main/examples/fortigateapi.py#L64
